cogs5e/sheetManager.py

```diff
@@ -393,7 +422,19 @@ async def character_list(self, ctx):
         ).to_list(None)
         if not user_characters:
             return await ctx.send('You have no characters.')
-        await ctx.send('Your characters:\n{}'.format(', '.join(sorted(c['name'] for c in user_characters))))
+        embed = embeds.EmbedWithAuthor(ctx)
+        embed.title = "Character List"
+        embed.description = ""
+        for char in sorted(user_characters,key=lambda k: k['name']):
+            charstring=f"{char.name} (`{char.upstream}`)\n"
+            newembed = embed
+            newembed.description+=charstring
+            if len(newembed) > 6000:
+                break
+            else:
+                embed = newembed
+        
+        await ctx.send(embed=embed)
```

Comment:
This should be a separate PR - I'm not sure that an embed with this information is the best format for this, and this PR is already wide enough in scope

Reply:
Fair enough, I just remember people saying to change it while I'm in here. I should stop doing that.